ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```diff
@@ -4,6 +4,18 @@ build_root:
     namespace: ci
     tag: "3.0"
   use_build_cache: true
+images:
+- dockerfile_literal: |
+    FROM quay.io/maistra-dev/maistra-builder:3.0
```

Comment:
this is wrong. It must be upstream image
https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci#L66